source/overview/product.rst

```diff
@@ -22,50 +22,53 @@ Core committers, including both community contributors and paid staff at Matterm
 Mattermost Editions
 -----------------------
 
-Mattermost is offered in three editions:
+Mattermost is offered in three editions, in multiple languages including U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, and Ukrainian. 
```

Comment:
@justinegeffen I'm curious why you feel it fits better here vs as part of Team Edition features?

Reply:
It was primarily aesthetic - the number of languages made the TE feature list list quite long and bulky. I also wasn't certain whether I wanted to add a link to the feature comparison or use "building on the features in Team Edition, E10" and so added it there as a blanket feature. 

I'm 2/5 on it though.

Reply:
Yeah, good point. My worry is if it makes the intro bulky/long with all the languages.

Reply:
I've given it another look and I agree with you; moved the languages back to TE section.